lib/parsers/tiff_parser.rb

```diff
@@ -43,5 +46,9 @@ def cr2?(io)
     safe_read(io, 2) == 'CR'
   end
 
+  def arw?(exif_data)
+    exif_data.compression == 6 && exif_data.new_subfile_type == 1 && exif_data.make == 'SONY'
```

Comment:
Did you learn this from what exiftool does?

Would be good to mention something about this in a comment, perhaps a link to the equivalent code in exifparser, or to some ARW spec document. Or otherwise a not explaining what `new_subfile_type == 1` means.

Reply:
I only read your comment in #159 later. Just having that or something similar as a comment in the code would already be valuable I think.

Reply:
I have added the exiftool code link since exifr was hard to understand 😆 https://github.com/remvee/exifr/blob/561bbd53b1e1999f60df73ad3e56e9303ed03424/lib/exifr/tiff.rb